delivery_carrier_label_paazl/data/delivery_carrier.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2020 Hunki Enterprises BV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="carrier_paazl" model="delivery.carrier">
+        <field name="name">Paazl</field>
+        <field name="code">PAZZL</field>
+        <field name="delivery_type">paazl</field>
+        <field name="integration_level">rate_and_ship</field>
+        <field name="product_id" ref="delivery.product_product_delivery" />
```

Comment:
Did you consider to add a separate product in each carrier module? Problem with delivery.carrier is that the company_id is a readwrite related field on the product so with multiple carriers for different companies it's easy to mess this up.

Reply:
I'm actually not sure if we shouldn't leave creating the carrier to the user entirely, and move this to demo. What do you think?